src/main/java/com/ounitech/wemove/models/Member.java

```diff
@@ -36,8 +36,8 @@ public class Member {
     @Column(name = "picture")
     private String picture;
 
-    @Column(name = "active", nullable = false, columnDefinition = "BYTE DEFAULT 0")
-    private Byte active;
+    @Column(name = "active", nullable = false, columnDefinition = "BOOLEAN DEFAULT 0")
+    private Boolean active;
```

Comment:
can we use just primitive type boolean ?
Boolean type means that we can have null values which is not the case here

Reply:
Done. Thanks for the feedback